### PR TITLE
feat: deprecate codeheroes, foosball, crm

### PR DESCRIPTION
**Pull request by mschilling**
Goal: extract all apps and libs other than Gamify, into its own space/repo;

Scope of work:

- [x] delete foosball app (including libs) - [Moved](https://github.com/dutchhackers/foosball-workspace)
- [x] delete codeheroes app (including libs) - [Moved](https://github.com/dutchhackers/codeheroes-workspace)
- [x] delete crm app (including libs) - [Moved](https://github.com/dutchhackers/crm-workspace)
- [x] update README

Next step:

This repo will be renamed into `move4mobile/gamify-workspace`

### CODE REVIEW

**Review by sanderelderman — CHANGES_REQUESTED** [review]
`libs/gamify/.gitignore` should not be deleted? This causes the `libs/gamify/data/generated` folder to be added to the staged files which is undesirable

**Review by cassshh — CHANGES_REQUESTED** [review]
There were 2 `.gitignore` files removed; `apps/- & libs/gamify`.
The one within `libs/...` is reintroduced, but not the same as before. As it still includes all generated stuff.

**Comment by mschilling** [comment]
> This causes the `libs/gamify/data/generated` folder to be added to the staged files which is undesirable

In doubt if this is really the case. (Will do some further checks..)

File: .github/codeql/codeql-config.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+name: "CodeQL config"
+paths-ignore:
+  - '**/generated'
```
Comment (mschilling) [inline .github/codeql/codeql-config.yml:3]: I don't want CodeQL scanner to scan this generated folder

**Comment by mschilling** [comment]
Merging now .. 🚀

**Pull request merged**